text/0225-configuration.md

```diff
@@ -0,0 +1,411 @@
+# OpenTelemetry Configuration
+
+A new configuration interface is proposed here in the form of a configuration model, which can be expressed as a file, and validated through a published schema.
+
+## Motivation
+
+OpenTelemetry specifies code that can operate in a variety of ways based on the end-user’s desired mode of operation. This requires a configuration interface be provided to the user so they are able to communicate this information. Currently, OpenTelemetry specifies this interface in the form of the API exposed by the SDKs and environment variables. This environment variable interface is limited in the structure of information it can communicate and the primitives it can support.
+
+### Environment Variable Interface Limitations
+
+The environment variable interface suffers from the following identified limitations:
```

Comment:
Also, environment variables have no versioning scheme, which prevents evolution over time.

Reply:
Added this to the list 👍🏻